packages/eslint-plugin/src/resolve-workspace-rules.ts

```diff
@@ -25,7 +25,7 @@ export const workspaceRules = ((): ESLintRules => {
     // Apply the namespace to the resolved rules
     const namespacedRules: ESLintRules = {};
     for (const [ruleName, ruleConfig] of Object.entries(rules as ESLintRules)) {
-      namespacedRules[`${WORKSPACE_RULE_NAMESPACE}/${ruleName}`] = ruleConfig;
+      namespacedRules[`${WORKSPACE_RULE_PREFIX}-${ruleName}`] = ruleConfig;
```

Comment:
Let's add it under both for now? That would make this change a lot less disruptive overall and I don't think cause any issues for either the majority of users using `.eslintrc.json`, nor the minority using flat config?

Reply:
We do have migration in place so I don't see this being any more disruptive than other code mods we do.

The failure in flat-config is correct - @nx/workspace/my-rule suggests there is a plugin called @nx/eslint-plugin-workspace with a rule named my-rule and config verification rightfully fails.

So far this is only happening for fat config but future consolidations could break this for standard config in between two patches. I would rather migrate to safe config now than once it breaks.

We anyway need to migrate existing flat config repos so why not do it for all of them?

Reply:
I didn't say not to do the migration, I only said to register it under both namespaces. I think we should do the migration but is there any downside to keeping the old namespace for now for folks who don't/can't run the migration straight away?